Directory.Build.props

```diff
@@ -18,7 +18,10 @@
         <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     </PropertyGroup>
 
-    <PropertyGroup>
+    <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+        <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    </PropertyGroup>>
+    <PropertyGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
Normally only this one alone should be enough

Reply:
@linkdotnet Fixed